std/algorithm/comparison.d

```diff
@@ -1194,7 +1194,15 @@ levenshteinDistanceAndPath(alias equals = "a == b", Range1, Range2)
 
 // max
 /**
-Returns the maximum of the passed-in values.
+This function iterates the passed arguments and returns the maximum value.
```

Comment:
Summary lines for functions should be imperative: "Iterate the passed arguments and return the maximum value."